--- docker-entrypoint.sh	2017-02-10 22:01:02.954643753 -0200
+++ modified_entrypoint.sh	2017-03-18 21:30:52.123441158 -0300
@@ -24,6 +24,8 @@
 }
 
 if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
+    source /host/secrets.sh
+
 	if ! [ -e index.php -a -e wp-includes/version.php ]; then
 		echo >&2 "WordPress not found in $PWD - copying now..."
 		if [ "$(ls -A)" ]; then
@@ -113,6 +115,7 @@
 if (isset($_SERVER['HTTP_X_FORWARDED_PROTO']) && $_SERVER['HTTP_X_FORWARDED_PROTO'] === 'https') {
 	$_SERVER['HTTPS'] = 'on';
 }
+define( 'WP_CONTENT_DIR', dirname(__FILE__) . 'wp-content' );
 
 EOPHP
 			chown www-data:www-data wp-config.php
@@ -145,6 +148,7 @@
 		set_config 'DB_USER' "$WORDPRESS_DB_USER"
 		set_config 'DB_PASSWORD' "$WORDPRESS_DB_PASSWORD"
 		set_config 'DB_NAME' "$WORDPRESS_DB_NAME"
+		set_config 'WP_CONTENT_DIR' "/var/www/wp-content"
 
 		for unique in "${uniqueEnvs[@]}"; do
 			uniqVar="WORDPRESS_$unique"
